src/org/labkey/test/tests/list/ListDateAndTimeTest.java

```diff
@@ -1101,9 +1102,9 @@ public void testInvalidDateAndTimeInsert() throws IOException, CommandException
 
         DataRegionTable regionTable = new DataRegionTable("query", getDriver());
         regionTable.clickInsertNewRow();
-        setFormElement(Locator.name("quf_" + timeCol), badTime);
-        setFormElement(Locator.name("quf_" + dateCol), badDate);
-        setFormElement(Locator.name("quf_" + dateTimeCol), badDateTime);
+        setFormElement(Locator.name(EscapeUtil.getFormFieldName(timeCol)), badTime);
+        setFormElement(Locator.name(EscapeUtil.getFormFieldName(dateCol)), badDate);
+        setFormElement(Locator.name(EscapeUtil.getFormFieldName(dateTimeCol)), badDateTime);
         clickButton("Submit");
```

Comment:
FYI. I was experimenting with Junie and had some pretty good results with these sorts of refactors.

"Update ListDateAndTimeTest to use UdateQueryRowPage" gave me this change:
```suggestion
        UpdateQueryRowPage updatePage = regionTable.clickInsertNewRow();
        updatePage.setField(timeCol, badTime);
        updatePage.setField(dateCol, badDate);
        updatePage.setField(dateTimeCol, badDateTime);
        updatePage.submit();
```
(LOL. Even with my typo in U~p~dateQueryRowPage, it worked.)

Reply:
I've followed through on a good number of these. There are still other usages that could make use of `QueryUpdateRowPage` but I'm not planning on modifying those now.